sql/expression/function/time.go

```diff
@@ -694,6 +701,75 @@ func (n *Now) WithChildren(children ...sql.Expression) (sql.Expression, error) {
 	return NewNow(children...)
 }
 
+// UTCTimestamp is a function that returns the current time.
+type UTCTimestamp struct {
+	precision *int
+}
+
+// NewUTCTimestamp returns a new UTCTimestamp node.
+func NewUTCTimestamp(args ...sql.Expression) (sql.Expression, error) {
+	var precision *int
+	if len(args) > 1 {
+		return nil, sql.ErrInvalidArgumentNumber.New("UTC_TIMESTAMP", 1, len(args))
+	} else if len(args) == 1 {
+		precisionArg, err := sql.Int32.Convert(args[0])
+
+		if err != nil {
+			return nil, err
+		}
+
+		n := int(precisionArg.(int32))
+		precision = &n
+	}
+
+	return &UTCTimestamp{precision}, nil
+}
+
+// Type implements the sql.Expression interface.
+func (ut *UTCTimestamp) Type() sql.Type {
+	return sql.Datetime
+}
+
+func (ut *UTCTimestamp) String() string {
+	if ut.precision == nil {
+		return "UTC_TIMESTAMP()"
```

Comment:
In `timediff` we're returning a lower case function name, but here it's upper case. Should choose one or the other.